server/controllers/user.controller.js

```diff
@@ -293,6 +293,21 @@ class UserController {
       return Response(res, 201, message);
     }
   }
+
+  /**
+     * @description Get list of users functionality
+     * @param {object} req
+     * @param {object} res
+     * @returns {object} List users functionality
+     */
+  static async getUsers(req, res) {
+    const users = await User.findAndCountAll({
+      attributes: {
+        exclude: ['password', 'istokenreset', 'subscribed']
```

Comment:
Good work excluding these attributes, can you also exclude the createdAt, updatedAt, roleId and social attributes in the user data? unless there is a reason to use them when getting all users data

Reply:
Ok I will implement but I think the `createdAt`  will be needed to sort the users list info on the frontend

Reply:
@geekdanitek is there is a need to sort the list of users in frontend? if sorting it should be by names

Reply:
Ok. Will implement feedback.

Reply:
Do not exclude `createdAt`. It could be used to sort by order of creation in ascending or descending order if we wish. Give room for that @geekdanitek @davidokonji

Reply:
Ok @paradisekelechi